README.md

```diff
@@ -30,112 +30,108 @@ To facilitate robust API governance, we're excited to introduce **API Analysis**
-- An API Center instance.
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio).
+- [Node.js 18.x](https://nodejs.org/en/download/releases/) or above. Use the `node --version` command to check your version.
+- [TypeScript 4.x](https://www.typescriptlang.org/). Use the `tsc -v` command to check your version.
 - [Visual Studio Code](https://code.visualstudio.com/) on one of the [supported platforms](https://code.visualstudio.com/docs/supporting/requirements#_platforms).
 - The [Azure Functions extension v1.10.4](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) or above for Visual Studio Code.
-- [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd)
-- [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli)
-
-### :cloud: How to deploy
-
-Follow the instructions in [Quickstart: Create a function in Azure with TypeScript using Visual Studio Code](https://learn.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-typescript?pivots=nodejs-model-v4#sign-in-to-azure). Start from the "Sign in to Azure" section and complete all subsequent sections.
+- [Azure Functions Core Tools v4.0.5382 or above](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Cisolated-process%2Cnode-v4%2Cpython-v2%2Chttp-trigger%2Ccontainer-apps&pivots=programming-language-typescript#install-the-azure-functions-core-tools).
 
-Alternatively, if you'd like to use [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd), run the following steps:
+### 🔧 Running the sample using the Azure Developer CLI (azd)
 
-1. Login to Azure by running the commands below in your terminal:
+The Azure Developer CLI (`azd`) is a developer-centric command-line interface (CLI) tool for creating Azure applications.
 
-    ```bash
-    az login
-    azd auth login
-    ```
+You need to install it before running and deploying with the Azure Developer CLI.
 
-1. Initialize your Azure environment by running the command below:
+### Windows
 
-    ```bash
-    azd init
-    ```
+```powershell
+powershell -ex AllSigned -c "Invoke-RestMethod 'https://aka.ms/install-azd.ps1' | Invoke-Expression"
+```
 
-   While you're running `azd init`, you will be asked to provide:
+### Linux/MacOS
 
-   - How to initialize: Choose the `Select a template` option, then `Y`.
-   - A template: Choose the `Minimal template` option. <!-- This option will be changed to "API Center Analyzer" once this PR is merged, which is not available at this time of PR -->
-   - An environment name: Any name you want to give to your environment.
+```
+curl -fsSL https://aka.ms/install-azd.sh | bash
+```
```

Comment:
There are many different ways of installing azd, even on Windows or MacOS. Can we instead provide the installation link, as a prerequisite?

Reply:
This is the link to point to: https://aka.ms/azd-install